src/main/webapp/index.html

```diff
@@ -80,7 +80,8 @@ <h3 class="panel__section-heading">Mail</h3>
             </a>
             <h3 class="panel__section-heading">Tasks</h3>
           </div>
-          <p class="panel__section-subheading">Selected Tasks:
+          <p class="panel__section-subheading">Selected Tasks: 
+            <select name="panel__task-list-select" id="panel__task-list-select" onchange="populateTasks()" size="2" multiple></select>
```

Comment:
Yep you can but then it would scale based on how many tasklists there are and would mess up the rest of the UI on the panel. Unfortunately, size 1 was a little too small to see what was going on.

Reply:
`RESOLVED`